rllib/offline/tests/test_dataset_reader.py

```diff
@@ -132,6 +132,15 @@ def test_dataset_shard_error_with_both_format_and_loader_fn(self):
         with self.assertRaises(ValueError):
             get_dataset_and_shards(config)
 
+    def test_default_ioctx(self):
+        # Test DatasetReader without passing in IOContext
+        input_config = {"format": "json", "paths": self.dset_path}
+        dataset, _ = get_dataset_and_shards(
+            {"input": "dataset", "input_config": input_config}
+        )
+        reader = DatasetReader(dataset)
+        reader.next()
```

Comment:
**Todo** in this test you need to check whether the actual batch size is 1 (the default). Rn it just checks whether the next() does not hit failure which is not just enough on its own.

Reply:
oops just merged...

Reply:
Done